app/ui/styles/components/form.css

```diff
@@ -22,10 +22,7 @@
   @apply space-y-6;
 }
 
-.ox-form.is-side-modal .ox-divider {
+.ox-form .ox-divider,
+.ox-form .ox-divider:not([hidden]) {
```

Comment:
I have this to avoid `!my-10` since the `space-y-...` applies it and is therefore more specific. Maybe `!` would be better? Unsure